plugins/colorschemes/nightfox.nix

```diff
@@ -14,6 +14,25 @@ helpers.neovim-plugin.mkNeovimPlugin config {
 
   maintainers = [ maintainers.GaetanLepage ];
 
+  colorscheme = null;
+  extraOptions = {
+    flavor = mkOption {
+      type = types.enum [
+        "carbonfox"
+        "dawnfox"
+        "dayfox"
+        "duskfox"
+        "nightfox"
+        "nordfox"
+        "terafox"
+      ];
+      example = "dayfox";
+      default = "nightfox";
+      description = "Which palette/flavor to use as the colorscheme.";
```

Comment:
Is it worth adding an "equivalent to doing xyz" note here or do you think that's overkill and/or more confusing than helpful?

Reply:
I would say that it would bring confusion.
When you think about it, most Nix modules are hierarchical: options most often only set the value of other options. In this sense, 90% of the options amount to "just setting this `foo` and `bar` options to 42".